src/main/perl/ComponentProxy.pm

```diff
@@ -139,6 +139,31 @@ sub getPostDependencies {
     return $self->{'POST_DEPS'};
 }
 
+=pod
+
+=item getFile()
+
+Returns the absolute filename of the components perl module.
+
+Receives an optional parameter with the base directory where the Perl
+module should be looked for.
+
+=cut
+
+sub getFile {
```

Comment:
Could this be called getComponentFilename instead? getFile suggests the return type could be a file handle rather than a string.